docs/HowTo/Interact/APIs/Authentication.md

```diff
@@ -91,15 +91,69 @@ for HTTP and WS.
+
+## JWT Public Key Authentication
+
+Enable authentication from the command line and supply the public key of the external JWT token. The JWT token must use the `RS256` algorithm.
+
+JWT public authentication disables the Besu `/login` endpoint, meaning [username and password authentication](username-and-password-authentication) will not work.
+
+### 1. Generate a Private and Public Key Pair
+
+The private and accompanying public key file must be in the `.pem` format.
+
+The key must use a RSA private key of at least 2048 bits.
+
+!!! example "Example using OpenSSL"
+    ```bash
+    openssl genrsa -out privateKey.pem 4096
+    openssl rsa -pubout -in privateKey.pem -pubout -out publicKey.pem
+    ```
+
+### 2. Create the JWT Token
+
+Create the JWT token using an external tool. 
+
+!!! important
+    The JWT token must use the `RS256` algorithm 
+
+Each payload for the JWT token must contain:
+
+* [JSON-RPC permissions](#json-rpc-permissions)
+* [`exp` (Expiration Time) claim](https://tools.ietf.org/html/rfc7519#section-4.1.4). 
+
+The following example uses the [JWT.io](https://jwt.io/) website to create a JWT token for testing purposes.
+
+![Create a JWT token](../../../images/JWT.png)
+
+### 3. Enable Authentication
+
+Use the [` --rpc-http-authentication-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-enabled) or 
+ [`--rpc-ws-authentication-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-ws-authentication-enabled)
+ options to require authentication for the JSON-RPC API.
+  
+Use the [`--rpc-http-authentication-jwt-public-key-file`](../../../Reference/CLI/CLI-Syntax.md#rpc-http-authentication-jwt-public-key-file) and [`--rpc-ws-authentication-jwt-public-key-file`](../../../Reference/CLI/CLI-Syntax.md#rpc-ws-authentication-jwt-public-key-file) to specify the public key to use with the externally created JWT token.
     
+## JSON-RPC Permissions 
+
+Each user has a list of permissions strings defining the methods they can access. To give access to: 
+
+* All API methods, specify `["*:*"]`.
+* All API methods in an API group, specify `["<api_group>:*"]`. For example, `["eth:*"]`. 
+* Specific API methods, specify `["<api_group>:<method_name>"]`. For example, `["admin:peers"]`.
+
+If authentication is enabled, to explicitly specify a user cannot access any methods, include the user with an empty permissions list (`[]`). 
+Users with an empty permissions list and users not included in the credentials file cannot access any JSON-RPC
+methods. 
+
 ## Using an Authentication Token to Make Requests 
 
 Specify the authentication token as a `Bearer` token in the JSON-RPC request header. 
 
 ### Postman
 
 In the _Authorization_ tab in the _TYPE_ drop-down list, select *Bearer Token* and specify the token 
-generated by the [`login` request](#obtaining-an-authentication-token). 
+generated [externally](#2-create-the-jwt-token), or by the [`login` request](#3-obtain-an-authentication-token). 
```

Comment:
I think it might be clearer to break this sentence into two - something like: 
In the _Authorization_ tab in the _TYPE_ drop-down list, select *Bearer Token* and specify the token 
generated. The token is generated [externally](#2-create-the-jwt-token) or by the [`login` request](#3-obtain-an-authentication-token).

Reply:
Done.